### PR TITLE
Comment out login with Facebook which is not working

### LINKED ISSUE #2509: Comment out "login with Facebook" which is not working
The FB login is not working. Also we have not tested and established the flow of how to add FB to an existing account. Please comment out the login with FB functionality for now.

Only allow login with Email for now.

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Comment out login with Facebook which is not working

#### Changes proposed in this pull request:

Fixes #2509

### CODE REVIEW

File: app/templates/components/forms/login-form.hbs [hunk 1]
```diff
@@ -49,15 +49,17 @@
...
-              {{#unless noSocial}}
-                <div class="ui horizontal divider">
-                  Or
-                </div>
-                <button type="button" class="ui facebook button" {{action 'auth' 'facebook'}}>
-                  <i class="facebook icon"></i>
-                  {{t 'Login with Facebook'}}
-                </button>
-              {{/unless}}
+              {{#if false}}
```
Comment (ritikamotwani) [inline app/templates/components/forms/login-form.hbs]: What is this?

Comment (harsh-jindal) [reply app/templates/components/forms/login-form.hbs]: @ritikamotwani I was getting template limiting error so I thought travis check might not pass. 
Screenshot 2019-04-07 at 8 15 31 PM
But I have made the changes.

**Comment by harsh-jindal** [comment]
@ritikamotwani Travis is failing due to commenting in code.Please suggest any other way of doing this.
Screenshot 2019-04-07 at 8 38 03 PM

**Comment by harsh-jindal** [comment]
@CosmicCoder96  @ritikamotwani I have used an if and put a false statement in it. As commenting is not allowed and it failed the travis check.

**Comment by kushthedude** [comment]
@harsh-jindal Commenting the code does not fails the build as there have been other issues and PR's too in which commenting was involved,  Please comment out the Connect With Facebook button from *setting/application*

File: app/templates/components/forms/login-form.hbs (hunk 1 above)
Comment (pradeepgangwar) [reply app/templates/components/forms/login-form.hbs]: @harsh-jindal This isn't what you should be doing. You must be doing it wrong. Please read this https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-html-comments.md. Try to find the solution to the errors before trying alternate solution.

**Comment by harsh-jindal** [comment]
@pradeepgangwar Thanks. From next time will keep that in mind.
@iamareebjamal Okay.

**Comment by CosmicCoder96** [comment]
@iamareebjamal @prateekj117 I set up a facebook test app for next.eventyay, stored the clientID and secret in settings, Facebook is able to authenticate, however post authentication it  tries to redirect to `localhost:8080`, what else needs to be configured? @prateekj117 Can you give a working demo>

**Comment by mariobehling** [comment]
@iamareebjamal We don't have a UI to connect/disconnect the login with FB. We also want to register people in the system rather than depending on FB. Furthermore Gnome, Nextcloud and others don't want Facebook. We need a way to configure this in the admin UI. This small feature requires too much focus now. We don't have time for this and need to focus on more urgent matters. People don't like FB so much anymore. We can have it, but we need to think everything through carefully in detail first. Therefore please merge this.

**Comment by pradeepgangwar** [comment]
@iamareebjamal This Pr was not the correct implementation 😅 . I requested changes on the PR. He added extra if else loop whereas issue was to just comment out the code.

**Comment by kushthedude** [comment]
@iamareebjamal @pradeepgangwar there were indentation erros too in the pr 😅 .
This should be reverted for now

**Comment by iamareebjamal** [comment]
@pradeepgangwar Please create a new PR on priority

**Comment by kushthedude** [comment]
SHould I do it @iamareebjamal there are two more places where it needs to be done

**Pull request merged**